cvmfs/receiver/payload_processor.cc

```diff
@@ -67,11 +67,15 @@ void PayloadProcessor::ConsumerEventCallback(
     path = event.object_name;
   } else {
     // kEmpty - this is an error.
-    LogCvmfs(kLogCvmfs, kLogStderr, "Event received with unknown object.");
+    LogCvmfs(kLogReceiver, kLogDebug | kLogSyslogErr,
+             "Event received with unknown object.");
     num_errors_++;
     return;
   }
 
+  LogCvmfs(kLogReceiver, kLogDebug | kLogSyslog,
```

Comment:
You may have a point here.

At the same time, here was my reasoning: Having this logged to syslog (which is the only logging being done in "production") would give me some way to diagnose things in case of problems. I also imagine that the syslog surely has log rotation enabled such that I don't fill up the root partition with this.

What do you think is the best approach?